pkg/plugins/resources/remote/unmarshalling.go

```diff
@@ -62,6 +62,7 @@ func UnmarshalList(b []byte, rs model.ResourceList) error {
 	if err := json.Unmarshal(b, rsr); err != nil {
 		return err
 	}
+	rs.GetPagination().SetTotal(rsr.ResourceList.Total)
```

Comment:
I think that if there is `next` it will override the pagination object and your total value is lost
```
rs.SetPagination(model.Pagination{
				NextOffset: offset,
			})
```

Reply:
Indeed. It turns out the interface is not very bulletproof. Shall we remove `SetPagination` and instead have `GetPagination().SetNext(..)`?

Reply:
I think SetPagination is slightly better since you are not modifying nested objects, but the whole object is so mutable it doesn't matter at this point. Let's got with `GetPagination().SetNext(..)`

Reply:
Done!